docs/en/ingest-management/elastic-agent/upgrade-elastic-agent.asciidoc

```diff
@@ -1,9 +1,64 @@
 [[upgrade-elastic-agent]]
 [role="xpack"]
-= Upgrade {agent}
+= Upgrade {stack} and {agent}
 
 experimental[]
 
+[discrete]
+[[upgrade-fleet]]
+== Upgrade to {fleet-server}
```

Comment:
I'd put this section at the bottom. People will have to upgrade their agents more often than upgrading FLeet server to 7.13. THat is a one time change.

Reply:
Thanks. I changed the heading to Upgrade from 7.12 or lower.